app/assets/javascripts/misc/pw-strength.js

```diff
@@ -55,6 +55,9 @@ function analyzePw() {
   const pwStrength = document.getElementById('pw-strength-txt');
   const pwFeedback = document.getElementById('pw-strength-feedback');
   const submit = document.querySelector('input[type="submit"]');
+  const forbiddenPasswords = document.getElementById(
+    'forbidden-passwords').getAttribute('data-forbidden-passwords'
+  );
```

Comment:
You can trim this back a bit by getting your element directly:
`document.querySelectorAll('[data-forbidden-passwords]')` if you may have more than one, or more likely `document.querySelector('[data-supported-jurisdictions]')` will work fine, if you know there will only be one.

When you have the element, you can work with the attribute though the `dataset` api, which I think will help you avoid a bunch of splitting below and you can just check if your value is in there (or not):  https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

Reply:
I can't get that to work. What I need to pass in to the zxcvbn method is an array, as in:
```js
zxcvbn(e.target.value, ['test@test.com', 'login.gov'])
```

This doesn't work:
```js
const forbiddenPasswords = document.querySelector('[data-forbidden-passwords]');

const z = zxcvbn(e.target.value, forbiddenPasswords.dataset['forbidden-passwords']);
```

Reply:
maybe something like this?

```
const pw_elem = document.querySelector('[data-forbidden-passwords]');
const z = zxcvbn(e.target.value, pw_elem.dataset.forbiddenPasswords.split(','))
```
The weird thing is that dataset is a set of all the data attrs on the elem, so you have to tell it which one you're working with. I always have to read the docs twice. :-D